docs/release-notes/8.10.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[release-notes-header-8.10.0]]
+== 8.10
+
+[discrete]
+[[release-notes-8.10.0]]
+=== 8.10.0
+
+[discrete]
+[[breaking-changes-8.10.0]]
+==== Breaking changes
+There are no breaking changes in 8.10.0.
+
+[discrete]
+[[features-8.10.0]]
+==== New features
+* Adds the MITRE ATT&CK® coverage page, which shows how the MITRE ATT&CK® adversary tactics and techniques are covered by your installed and enabled detection rules ({pull}161556[#161556], {pull}163498[#163498], {pull}164613[#164613], {pull}164986[#164986]).
+* Adds a component under the Elastic AI Assistant header that allows you to select a different connector ({pull}163666[#163666]).
+* Adds a flyout that allows you to examine rule details when installing or updating a prebuilt rule ({pull}163304[#163304]).
+* Adds the ability to specify custom highlighted fields for an alert ({pull}163235[#163235]).
+* Adds role-based access control privileges for the Elastic AI Assistant ({pull}163031[#163031]).
+* Adds the **Reputation service** option to the malicious behavior protection setting on the Elastic Defend integration policy ({pull}161617[#161617]).
+
+[discrete]
+[[enhancements-8.10.0]]
+==== Enhancements
+* Modifies the {elastic-sec} main menu by adding the Rules main page and landing page, and shifting the order of the Cases and Explore pages ({pull}165061[#165061], {pull}163102[#163102], {pull}161667[#161667]).
+* Adds the `resource.id` column to top ten vulnerability resources ({pull}162668[#162668]).
+* Findings tab redirection logic ({pull}162289[#162289]).
```

Comment:
```suggestion
* Shows the most relevant tab when you open the Findings page, instead of always showing the Misconfigurations tab ({pull}162289[#162289]).
```